sql/databases.go

```diff
@@ -58,7 +58,7 @@ type CollatedDatabaseProvider interface {
 // always) implemented by a DatabaseProvider.
 type TableFunctionProvider interface {
 	// TableFunction returns the table function with the name provided, case-insensitive
```

Comment:
(minor) wouldn't hurt to mention the boolean return param in the docs here and state that it means the table function was found. It's kinda obvious from other patterns in our code base, but it's a nice touch to explicitly call it out in the godocs here.